server/utils/programmeUtils.test.ts

```diff
@@ -0,0 +1,31 @@
+import { programmeFactory, programmePrerequisiteFactory } from '../testutils/factories'
+import { prerequisiteSummaryListRows, programmeListItems } from './programmeUtils'
+
+describe('programmeUtils', () => {
+  describe('prerequisiteSummaryListRows', () => {
+    it('converts prerequisites into summary list rows', () => {
+      const prerequisites = programmePrerequisiteFactory.buildList(3)
+
+      const expected = prerequisites.map(prerequisite => {
+        return { key: { text: prerequisite.key }, value: { text: prerequisite.value } }
```

Comment:
Can we do the text formatting/capitalisation here? It _could_ be the responsibility of the API, but I think it's safest to do this in the UI so we can be sure it'll always look consistent.

Reply:
We could, but then would we not also want to do that with every text value that comes from the API, e.g. the title, description etc, if we can't rely on the API giving us clean data?

Reply:
That's a good point. I suppose as we're in control of the API we can just ensure it comes back in the right format. When using other APIs we may need to revisit this, but I think I'm happy to sort this on the API side.